WooCommerce/Classes/ViewRelated/Upgrades/OwnerUpgradesView.swift

```diff
@@ -3,15 +3,19 @@ import Yosemite
 import WooFoundation
 
 struct OwnerUpgradesView: View {
+    @ObservedObject var subscriptionsViewModel: SubscriptionsViewModel
```

Comment:
I don't love the dependency on the `SubscriptionsViewModel`... I know we had it already, but this is binding it more and deeper in the view hierarchy. It's not great to be using an unrelated view's view model here, and this might be a good time to refactor it.

If we need to bring the `CurrentPlanDetailsView` inside the `OwnerUpgradesView`, perhaps we could have a `CurrentPlanDetailsViewModel`, or just some State/Binding properties?

Reply:
I had my reservations as well about this change, specially since we still need to refactor other models, but the webview is still being used. I moved ahead with binding the properties for the time being on 0b863502a57e016042b45b722e4cc372d3f47f31 to follow the same approach we're doing with `isLoading` and this way we don't tie the `SubscriptionsViewModel` here. Happy to iterate further!